packages/mobile/android/app/src/main/AndroidManifest.xml

```diff
@@ -81,6 +81,11 @@
                 android:resource="@xml/file_paths"
             />
         </provider>
+        <provider
```

Comment:
awesome find! how did you figure this out?

Reply:
Was getting `Cannot create URL for blob` error and found this issue https://github.com/charpeni/react-native-url-polyfill/issues/284